### PR TITLE
MM-14318 centerChannelStyles spec

### PR DESCRIPTION
**Pull request by m3phistopheles**
#### Summary
Test Spec for Center Channel Styles, under custom themes in Mattermost settings.

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/10367

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Ran `make check-style` to check for style errors (required for all pull requests)
- [X] Ran `make test` to ensure unit and component tests passed

### CODE REVIEW

**Comment by m3phistopheles** [comment]
Removed post-save assertions for Mention Highlight BG and Link Color tests. Cypress fails to save these settings, causing the final color assertion to fail. These tests pass manually, but until there's a fix from cypress, we can only make color changing assertions before saving.

File: cypress/integration/account_settings/theme_color/custom_colors_centerChannelStyles_spec.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```
Comment (saturninoabril) [inline cypress/integration/account_settings/theme_color/custom_colors_centerChannelStyles_spec.js]: Please change filename to `custom_colors_center_channel_styles_spec.js`

**Comment by jasonblais** [comment]
Hey @m3phistopheles, touching base to hear if you had any questions about the feedback above?

**Comment by hanzei** [comment]
Hey @m3phistopheles,

Are you still working on this issue? Can we help with something?

Please note that we will close this PR after another 10 days of inactivity. If you don't object, other contributors might complete this PR for you.

**Comment by hanzei** [comment]
Same note as on https://github.com/mattermost/mattermost-webapp/pull/2514. Thanks for this PR, but I will close this one due to inactivity.

**Pull request closed without merging**